src/utils/buffer.test.ts

```diff
@@ -43,6 +70,7 @@ describe('buffer', () => {
     expect(await timingSafeEqual({ a: 1 }, { a: 2 })).toBe(false)
     expect(await timingSafeEqual([1, 2], [1, 2])).toBe(false)
     expect(await timingSafeEqual([1, 2], [1, 2, 3])).toBe(false)
+    expect(await timingSafeEqual('a', 'b', () => undefined)).toBe(false)
```

Comment:
Test covering the first if statement branch

https://github.com/honojs/hono/blob/7ee829be6bef89746a0157c007ea9668141ddce9/src/utils/buffer.ts#L41-L43